docker-compose.yml

```diff
@@ -31,24 +31,18 @@ services:
       - APP_INSIGHTS_INSTRUMENTATION_KEY
     ports:
       - "8081:8081"
-    volumes:
-      - ./pccommon:/opt/src/pccommon
-      - ./pcstac:/opt/src/pcstac
     depends_on:
       - database
     command: >
       bash -c "pypgstac pgready && uvicorn pcstac.main:app --host 0.0.0.0 --port 8081 --reload --proxy-headers"
   tiler:
     image: pc-apis-tiler
+    platform: linux/amd64
```

Comment:
make sure anyone can build the docker image. Rasterio wheels do not exist for `amd64` platform

Reply:
This requires turning on the "experimental": "true" flag for Docker Engine on windows machines. Because this is accommodating folks M1 machines and would break existing windows setups without that flag set, I'd prefer to keep this as a note in the README rather than in the docker compose.

Reply:
good point